tests/bluetooth/df/connectionless_cte_chains/src/test_remove_cte_from_chain.c

```diff
@@ -33,9 +34,11 @@
 #include "common.h"
 
 #define TEST_ADV_SET_HANDLE 0
+#define TEST_CTE_COUNT 3
 #define TEST_PER_ADV_CHAIN_LENGTH 5
+#define TEST_PER_ADV_CHAIN_INCREASED_LENGTH 7
+#define TEST_PER_ADV_CHAIN_DECREASED_LENGTH (TEST_CTE_COUNT - 1)
```

Comment:
Should this have been like this? 

```suggestion
#define TEST_PER_ADV_CHAIN_INCREASED_LENGTH 7
#define TEST_PER_ADV_CHAIN_DECREASED_LENGTH (TEST_PER_ADV_CHAIN_LENGTH - 1)
```

Or what is the relation between length and CTE count?

Reply:
`TEST_CTE_COUNT` is also used as chain length. It depends if already created chain is longer or shorter than requested number of CTEs. In case of `TEST_PER_ADV_CHAIN_DECREASED_LENGTH` it is deliberately set to length smaller than CTE number to see if there is no problem with release of advertising PDUs while disabling CTE.